docker-compose.override.yml

```diff
@@ -7,6 +7,7 @@ services:
   nginx:
     volumes:
       - mediawiki-volume:/srv/mediawiki
+      - static-volume:/srv/static
```

Comment:
I think this belongs to the prod configuration - default and override should only contain config for development.